src/extension.ts

```diff
@@ -53,10 +53,19 @@ export function activate(context: vscode.ExtensionContext) {
     //register a link provider for this extension's "BrightScript Log" output
     vscode.languages.registerDocumentLinkProvider({ language: 'Log' }, docLinkProvider);
     //give the launch config to the link provder any time we launch the app
-    vscode.debug.onDidReceiveDebugSessionCustomEvent((e) => {
+    vscode.debug.onDidReceiveDebugSessionCustomEvent(async (e) => {
         if (e.event === 'BSLaunchStartEvent') {
             docLinkProvider.setLaunchConfig(e.body);
             logOutputManager.setLaunchConfig(e.body);
+        } else if (!e.event) {
```

Comment:
What happens when `e.event` is truthy, but is not `"BSLaunchStartEvent"`?

Reply:
In that use case it could be ether a `BSLogOutputEvent` or a `BSRendezvousEvent` once once #169  is merged.